Makefile

```diff
@@ -0,0 +1,17 @@
+test:
+	poetry run pytest ./tests
```

Comment:
Is this a leftover from the main repo? Would customers ever need to run this on their config repo?

Reply:
The idea here was to set it up so users could chose if they wanted to use the `pytest` route or the `pypanther test` route. And see the example of using either. Plus this shows off how users can use regular python testing.